app/assets/stylesheets/govuk_publishing_components/components/_layout-header.scss

```diff
@@ -28,6 +28,63 @@
   }
 }
 
+.gem-c-layout-header--search-left {
+
+  .gem-c-header__menu-button {
```

Comment:
This button doesn't seem to be working in the component guide - is that just a quirk of the component guide or is something broken?

Reply:
As of right now, it seems to work as expected for me! Perhaps it used to be broken and has been fixed in the meanwhile. 
Will be marking this as resolved but feel free to "unresolve" if there is still an issue for you.